docs/rum_collection/web_view_tracking.md

```diff
@@ -0,0 +1,62 @@
+---
+beta: true
+dependencies: 
+- https://github.com/DataDog/dd-sdk-ios/blob/master/docs/web_view_tracking.md
+title: iOS Web View Tracking
+kind: documentation
+description: Monitor web views in your hybrid iOS applications.
+further_reading:
+- link: "/real_user_monitoring/ios/"
+  tag: "Documentation"
+  text: "iOS Monitoring"
+- link: "/real_user_monitoring/browser/"
+  tag: "Documentation"
+  text: "Browser Monitoring"
+---
+
+## Overview
+
+Real User Monitoring allows you to monitor web views and eliminate blind spots in your hybrid iOS and tvOS applications.
+
+You can:
+
+- Track user journeys across web and native components in mobile applications
+- Scope the root cause of latency to web pages or native components in mobile applications
+- Support users that have difficulty loading web pages on mobile devices
+
+## Setup
+
+### Prerequisites
+
+Set up the web page you want rendered on your mobile iOS and tvOS application with the Browser SDK first. For more information, see [RUM Browser Monitoring][1].
+
+### Update your existing SDK setup
+
+1. Download the [latest version][2] of the RUM iOS SDK.
+2. Edit your existing iOS SDK setup from [RUM iOS Monitoring][3].
+3. Add tracking for web views with the following example:
+
+   ```
+   // Start tracking
+   webView.configuration.userContentController.trackDatadogEvents(in: hosts)
+   // Stop tracking
+   // Note: This method must be called when the webview is de-initialized.
+   webView.configuration.userContentController.stopTrackingDatadogEvents()
+   ```
```

Comment:
Unlike in Android SDK, the iOS APIs for webview tracking are not part of the SDK setup which we link to in `[3]`. Instead, we provide the two APIs from the snippet. IMO, we should change this section from _"Update your existing SDK setup"_, to something like _"Instrument web view with Datadog"_. We could use markdown to list and explain each API like we do [here](https://docs.datadoghq.com/real_user_monitoring/ios/advanced_configuration/?tab=swift#rum-configuration).

I can propose something like this 🙂:

> ### Instrument your web views
> 
> The latest version of the RUM iOS SDK comes with two APIs, which let you control web view tracking. Both are declared as an extension to [`WKUserContentController`](https://developer.apple.com/documentation/webkit/wkusercontentcontroller):
>
> `trackDatadogEvents(in hosts: Set<String>)`
> : Enables tracking RUM events in this web view for particular `hosts`.
>
> `stopTrackingDatadogEvents()`
> : Disables tracking for this web view. This should be typically called when you are done with given web view and it is about to be deallocated.
>
> Example:
> ```swift
> import WebKit
> import Datadog
>  
> webView.configuration.userContentController.trackDatadogEvents(in: ["example.com"])
> ```

WDYT?